Adding PHP Dynamic instrumentation preview details to docs (#27963)
DataDog/documentation: content/en/dynamic_instrumentation/enabling/php.md

See [Dynamic Instrumentation][5] for information about adding instrumentations and browsing and indexing the data.

> ## Limitations

Reviewer: ```suggestion
```

I would suggest to remove this as it's a little confusing with a heading "limitations" followed by a heading of Supported Features, as a reader it's unclear

Reviewer 2: This goes against the design of [Node.js](https://docs.datadoghq.com/dynamic_instrumentation/enabling/nodejs/#limitations) and [Ruby](https://docs.datadoghq.com/dynamic_instrumentation/enabling/ruby/#limitations) pages, which both have a "Limitations" section.

I don't personally think this headline is a problem, and I like the ability to deep link to it which can come in handy in support cases. It's also nice to have it show up the the ToC.

I see that we have the following options:
1. Keep the headline (I don't find it confusing)
2. Rename the headline, so it's less confusion
3. Change the wording of the sentence following the headline, so it's less confusion
4. Remove the headline in Node.js and Ruby as well

Reviewer: Let me know what you all decide and if you need a re-review after :)

Author: @watson Good points about the deep link and ToC. I'll revert the change back to "Limitations" and we can think about a better headline for this section and apply it across all the docs in a separate PR (if we come up with one).

> The following limitations apply to the limited preview:

Reviewer: ```suggestion
<div class="alert alert-info">The following features are supported in the Limited Preview</div>
```

I would suggest to change this to an alert so it's more clear.

### Supported features

- Dynamic Logs, Metrics, Spans, and Span Tags
- Local variable capture for Dynamic Logs
- Dynamic Log conditions
- Expression evaluation in Dynamic Log templates

> - [PII redaction][8] based on variable/property names

Author: ```suggestion
- [PII redaction][8] based on variable/property names and classes
```

- [Source code integration][9]

### Unsupported features

- Instrumentation attached to a specific file/line

> - PII redaction based on specific classes or types

Reviewer 3: PII redaction is supported.

> - PII redaction based on specific classes or types

Author: ```suggestion
```

See [Dynamic Instrumentation][5] for information about adding instrumentations and browsing and indexing the data.

## Limitations

The following limitations apply to the limited preview:

### Supported features

- Dynamic Logs, Metrics, Spans, and Span Tags

> - Dynamic Logs attached to a function/method

Reviewer 4: we only mention function/method for attached logs. but I assume this is for all probe types? 

If so, this line and line "Dynamic Logs attached to a specific file/line" at the unsupported features is confusing.

Author: @shatzi Let me know if the updated version is better!

- Local variable capture for Dynamic Logs
- Dynamic Log conditions
- Expression evaluation in Dynamic Log templates
- [PII redaction][8] based on variable/property names
- [Source code integration][9]

### Unsupported features

- Dynamic Logs attached to a specific file/line
- PII redaction based on specific classes or types

> - Source code symbol autocomplete and search

Reviewer 2: We don't mention this on the Ruby and Node.js page. Maybe just delete it here?

Author: @watson Removed this line!
